### PR TITLE
tmux_env.py: support python 3.x and fixes

### PR DESCRIPTION
**Pull request by dobbymoodge**
This commit replaces a `dict.iteritems()` call with the correct
`dict.items()` call. It also adds the `text=True` arg to the
`subprocess.check_output` call so the output can be handled as a
`str`. Other changes include minor formatting tweaks and replacing a
`+`-style string concatenation with "f-string" style string
formatting.

These changes most likely break Python 2.x compatibility

### CODE REVIEW

**Comment by dobbymoodge** [comment]
Work-in-progress, pending review and acceptance of upstream PR: https://github.com/agriffis/weechat-tmux-env/pull/3

**Comment by dobbymoodge** [comment]
The script author tells me he doesn't maintain it anymore and would prefer this repository be canonical for this script; see this PR comment: https://github.com/agriffis/weechat-tmux-env/pull/3#issuecomment-571795006

Upstream review/merge is no longer needed, so please review this change for merging.

**Comment by flashcode** [comment]
Hi @dobbymoodge, please ensure that Python 2 is still supported by the script (I moved the end date for Python 3 migration, so for now all scripts must still be compatible with Python 2).
Once compatibility is OK, please remove the comment in script.

**Comment by dobbymoodge** [comment]
This works with Python 2 and 3 now.

File: python/tmux_env.py [hunk 1]
```diff
@@ -84,18 +92,31 @@ def config_cb(data, option, value):
...
     """Check if tmux is attached, update environment"""
 
-    attached = os.access(SOCK, os.X_OK) # X bit indicates attached
+    attached = os.access(SOCK, os.X_OK)  # X bit indicates attached
     if attached:
         update_environment()
     return w.WEECHAT_RC_OK
 
+
+def do_check_output(command_args):
```
Comment (dobbymoodge) [inline python/tmux_env.py]: @flashcode If you know of some other way to support methods with different function signatures between Python 2 and 3, I'd be happy to replace this with something better. :)

**Comment by dobbymoodge** [comment]
@flashcode Any chance of a follow-up review for this? It's been running stable on my workstation for a couple of weeks now.

**Comment by flashcode** [comment]
Thanks, merged.

**Pull request closed without merging**